### PR TITLE
Include Prometheus exporter into ROCK

### PR DESCRIPTION
**Pull request by gtrkiller**
Applicable spec: <link>

### Overview

Install the statsd Prometheus exporter into gunicorn's ROCK

### Rationale

This change simplifies the architecture of the charm by reducing the amount of resources and containers it needs

### Juju Events Changes

N/A

### Module Changes

N/A

### Library Changes

N/A

### Checklist

- [X] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [X] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [X] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [X] The documentation is generated using `src-docs`
- [X] The documentation for charmhub is updated
- [X] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: .trivyignore [hunk 1]
```diff
@@ -1 +1,2 @@
 CVE-2022-40897
+CVE-2023-39325
```
Comment (arturo-seijas) [inline .trivyignore:2]: Can you double check the .trivyignore file and remove whatever's not needed anymore?

File: gunicorn_rock/rockcraft.yaml [hunk 2]
```diff
@@ -19,6 +19,8 @@ parts:
     - coreutils
     - gunicorn
     - python3-psycopg2
+    stage-snaps:
+    - gtrkiller-statsd-prometheus-exporter/latest/edge
```
Comment (arturo-seijas) [inline gunicorn_rock/rockcraft.yaml:23]: Can you publish this with a generic prefix/suffix instead of "gtrkiller"?

File: .trivyignore (hunk 1 above)
Comment (gtrkiller) [reply .trivyignore:2]: Already double checked, both are necessary

Comment (merkata) [reply .trivyignore:2]: How did you check - is there a run without them that fails? The action might have a bug then..

**Review by merkata — DISMISSED** [review]

File: .trivyignore (hunk 1 above)
Comment (gtrkiller) [reply .trivyignore:2]: Yes, I checked with the following runs: https://github.com/canonical/gunicorn-k8s-operator/actions/runs/7557994564/job/20578506931  https://github.com/canonical/gunicorn-k8s-operator/actions/runs/7557890775/job/20578171237

**Review by nrobinaubertin — APPROVED** [review]

**Pull request merged**